spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -205,6 +226,10 @@ def tmp2ofolder(self):
             copy(os.path.join(self.tmp_dir, file_), os.path.join(self.path_ofolder, file_))
 
     def pack_measures(self):
+
+        # Replace 'nan' with 'n/a' ('nan' results in empty cells in Excel)
+        # self.measure_pd = self.measure_pd.replace(np.nan, 'n/a')
+
```

Comment:
Why is there some commented-out code here? Should it be removed?

Reply:
good catch!
this is legacy of https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4489#discussion_r1670859216 and https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4489/commits/1d4641408b269d71e27c59177fe64b2aac38ec29
We can remove these lines